### ISSUE TITLE
kf.newstuff.core errors during global theme installation

### ISSUE THREAD

**Issue by tnyeanderson** [Mrcuve0/Aritim-Dark#22]
**Describe the bug**
Get errors when trying to install the KDE global theme portion of this package. Other parts install with no issue. Please see below for steps to reproduce.

**To Reproduce**
Using OpenSUSE tumbleweed:
```bash
# Install themes and color schemes, etc
curl -Lo '/tmp/aritim-dark.zip' 'https://github.com/Mrcuve0/Aritim-Dark/archive/refs/heads/master.zip'
unzip -q -d /tmp /tmp/aritim-dark.zip
kpackagetool5 -i /tmp/Aritim-Dark-master/KDE/auroraeTheme
kpackagetool5 -i /tmp/Aritim-Dark-master/KDE/globalTheme # <-- THIS IS WHERE THE ISSUE HAPPENS
kpackagetool5 -i /tmp/Aritim-Dark-master/KDE/plasmaTheme/Aritim-Dark-Flat-Blur
kpackagetool5 -i /tmp/Aritim-Dark-master/KDE/plasmaTheme/Aritim-Dark-Flat-Solid
cp "/tmp/Aritim-Dark-master/KDE/colorScheme/AritimDark.colors" "$HOME/.local/share/color-schemes/"
cp "/tmp/Aritim-Dark-master/KDE/konsole/Aritim Dark.colorscheme" "$HOME/.local/share/konsole/"
rm -rf "/tmp/aritim-dark.zip" "/tmp/Aritim-Dark-master"
```

See screenshots below for the error messages. My apologies I cannot paste the text as this is being tested in a VM.

**Expected behavior**
I expect to see `Successfully installed /tmp/Aritim-Dark-master/KDE/globalTheme` with no error message.

**Screenshots**

![Screenshot-2021-12-07-00 01 15](https://user-images.githubusercontent.com/9835153/144969489-55f81ae6-0a69-4c11-9852-c8f2f61a98eb.png)

**Desktop (additional informations):**
 - OS: OpenSUSE tumbleweed
 - KDE Plasma Version: 5.23.4
 - KDE Frameworks Version: 5.88.0
 - Qt version: 5.15.2

**Additional context**
Might be related: #1 , #4 , [Aritim-Light#2](https://github.com/Mrcuve0/Aritim-Light/issues/2)

Thanks for the theme! It's the best out there :)

**Comment by Mrcuve0** [comment]
Thank you for feedback!

Just let me add a FYI comment on what you did: in the first terminal snippet, instead of installing every plasma component (plasmatheme, globaltheme, colorscheme etc) sequentially, you can simply install the globalTheme package and everything will be installed automatically for you. Additionally, I suggest you downloading the global theme directly from the KDE Store: that package is the first being updated, the one in the "release" page here in GitHub is not as updated as that one (I know, i simply keep forgotting updating it, sorry).

Now, let's discuss the problem you highlighted. I'm aware of the error messages and, at least on my side, these do not prevent the actual installation of the theme (let me know if everything got installed correctly!). For instance, installing the theme from the settings GUI should not (at least in my case) throw any error and everything should be installed correctly.

In any case, thank you for reporting this issue! I'll try to solve it in the upcoming holidays.
Have a nice day!

**Comment by tnyeanderson** [comment]
You are absolutely right that it doesn't seem to prevent the actual installation of the theme. I will try installing only the `globalTheme` package to see if it installs all the pieces.

Unfortunately I cannot use the KDE store as you suggest, as this is going into a script for automation.

Thanks for taking a look, and you have a nice day too :)
